src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/Bau/BauCacheController.cs

```diff
@@ -179,6 +199,20 @@ public class ClearPublicCacheReleasePathsViewModel
             public HashSet<string> Paths { get; set; } = new();
         }
 
+        public class ClearPublicCachePublicationPathsViewModel
```

Comment:
Going with our current naming convention this would be `ClearPublicCachePublicationPathsRequest` or even `DeletePublicCachePublicationPathsRequest` and could be a record type?